bundles/notion-proxy-rest-api/src/FondOfOryx/Glue/NotionProxyRestApi/NotionProxyRestApiFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace FondOfOryx\Glue\NotionProxyRestApi;
+
+use FondOfOryx\Glue\NotionProxyRestApi\Processor\Builder\RestResponseBuilder;
+use FondOfOryx\Glue\NotionProxyRestApi\Processor\Builder\RestResponseBuilderInterface;
+use FondOfOryx\Glue\NotionProxyRestApi\Processor\Creator\NotionProxyRequestCreator;
+use FondOfOryx\Glue\NotionProxyRestApi\Processor\Creator\NotionProxyRequestCreatorInterface;
+use Spryker\Glue\Kernel\AbstractFactory;
+
+/**
+ * @method \FondOfOryx\Client\NotionProxyRestApi\NotionProxyRestApiClientInterface getClient()
+ */
+class NotionProxyRestApiFactory extends AbstractFactory
+{
+    /**
+     * @return \FondOfOryx\Glue\NotionProxyRestApi\Processor\Creator\NotionProxyRequestCreatorInterface
+     */
+    public function createNotionRequestCreator(): NotionProxyRequestCreatorInterface
```

Comment:
createNotionProxyRequestCreator maybe?

Reply:
thx for the advise @julianzimmermann i made the adjusments